app/stylesheet/application-webpack.scss

```diff
@@ -16,4 +16,5 @@
 @import './notifications.scss';
 @import './quadicon.scss';
 @import './tree.scss';
+@import './toastnotification.scss';
```

Comment:
Can you add css in notifications.scss file instead of creating new file for few lines of css?